tests/unit/test_subprocpool.py

```diff
@@ -197,3 +203,98 @@ def test_xfunction_attribute_error(self):
 
 if __name__ == '__main__':
     unittest.main()
+
+
+@pytest.fixture
+def mock_ctx():
+    def inner_(ret_code=None, host=None, cmd_key=None):
+        """Provide a SimpleNamespace which looks like a ctx object.
+        """
+        inputs = locals()
+        defaults = {
+            'ret_code': 255, 'host': 'mouse', 'cmd_key': 'my-command'
+        }
+        for key in inputs:
+            if inputs[key] == None:
```

Comment:
Take or leave the suggestions in this file. They are auto generated from pycodestyle...
```suggestion
            if inputs[key] is None:
```